release notes/v0.47.0.md

```diff
@@ -75,12 +77,21 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 - [browser#1040](https://github.com/grafana/xk6-browser/pull/1040) fix the `context.clearCookies()` method to clear all cookies from the current browser context.
 - [browser#1031](https://github.com/grafana/xk6-browser/pull/1031) fix the `expires` field while adding cookies using the `context.addCookie()` method.
 - [browser#1039](https://github.com/grafana/xk6-browser/pull/1039) fix Goja conversions while adding and retrieving cookies.
+- [browser#1038](https://github.com/grafana/xk6-browser/pull/1038) Fix read/write race condition for edge case with remote browsers.
+- [browser#1034](https://github.com/grafana/xk6-browser/pull/1034) Fix `page.reload` & `page.setContent` to use the default navigation timeout over the default timeout.
+- [browser#1033](https://github.com/grafana/xk6-browser/pull/1033) Fix the `page` timeouts so it is actually used after being set.
+- [browser#1022](https://github.com/grafana/xk6-browser/pull/1022) Fix the `check` in examples/fillform.js so that it matches the type definitions and documentation for `check`.
```

Comment:
I'm not sure about this one.
```suggestion
- [browser#1022](https://github.com/grafana/xk6-browser/pull/1022) Fix the `check` in `examples/fillform.js` so that it matches the type definitions and documentation for `check`.
```